### DISCUSSION TITLE
视频下载页面只有toast选项，没有aria2导出

### DISCUSSION THREAD

**Discussion by Lcdan0220** [the1812/Bilibili-Evolved#2752] [提问 / Q&A]
求助，手动安装了 视频下载功能和视频下载aria2支持功能，aria2客户端也正常运作监听，但是输出方式只能选toast，没有以前的aria2导出选项。

**Comment by the1812** [comment]
手动? 那确定是装到插件里了吗, aria2 要装在插件里 (不是组件)
